build.sh

```diff
@@ -1,7 +1,19 @@
-git clone https:\/\/github.com\/fossasia\/fossasia.org.git out
+git config --global push.default simple
+ssh-agent bash -c "ssh-add ${travis_deploy_key}"
```

Comment:
To save the key to a file, do 

    echo "$travis_deploy_key" > id_rsa

To execute a git command, with the key, do 

    ssh-agent bash -c "ssh-add id_rsa; GIT COMMAND"